Mathlib/RingTheory/Ideal/QuotientOperations.lean

```diff
@@ -124,16 +124,13 @@ variable {R : Type u} {S : Type v} {F : Type w} [CommRing R] [Semiring S]
 theorem map_quotient_self (I : Ideal R) : map (Quotient.mk I) I = ⊥ :=
   eq_bot_iff.2 <|
     Ideal.map_le_iff_le_comap.2 fun _ hx =>
-    -- Porting note: Lean can't infer `Module (R ⧸ I) (R ⧸ I)` on its own
-      (@Submodule.mem_bot (R ⧸ I) _ _ _ Semiring.toModule _).2 <|
-          Ideal.Quotient.eq_zero_iff_mem.2 hx
+      (Submodule.mem_bot (R ⧸ I) ..).2 <| Ideal.Quotient.eq_zero_iff_mem.2 hx
```

Comment:
nit: I think using `_` or `_ _` is probably clearer than `..` here.

Reply:
Today I learned that `..` accepts `0` arguments